python/reg_interface/scripts/repeated_reg_read.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+
+from reg_utils.reg_interface.common.reg_xml_parser import getNode, parseXML
+from reg_utils.reg_interface.common.reg_base_ops import readAddress
+from reg_utils.reg_interface.common.reg_base_ops import writeReg
+from reg_utils.reg_interface.common.print_utils import printRed
+from reg_utils.reg_interface.common.jtag import initJtagRegAddrs
+from reg_utils.reg_interface.common.reg_base_ops import *
+
+#from rw_reg import *
+#from mcs import *
+
+import time
+import array
+import struct
+import os
+import sys
+import socket
+
+import argparse
+
+if __name__ == '__main__':
+
+    from optparse import OptionParser
+    parser = OptionParser()
+    parser.add_option("-f", "--filename", type="str", dest="filename", help="Filename to which output information is written", default="repeated_reg_read.txt")
+
+    (options, args) = parser.parse_args()
+
+    if len(args) != 3:
+            print('Usage: repeated_reg_read.py <register_name> <number of times> <sleep time in microseconds>')
+            exit(os.EX_USAGE)
+
+    f = open(options.filename,"w")        
+            
+    parseXML()
+    
+    hostname = socket.gethostname()
+
+    if hostname == "eagle26":
+        print "This script is running on eagle26."
+        pass
+    else:
+       print "This script is not running on eagle26. rpc_connect(\"eagle26\") will be called."
+       rpc_connect("eagle26")   
+
+       HW_RW_REG 
+
+    writeReg(getNode("GEM_AMC.GEM_SYSTEM.CTRL.LINK_RESET"),0x1)   
+
+    time.sleep(float(sys.argv[3])/float(1000000))
```

Comment:
Why are you using `sys.argv` container when you have created an `OptionParser()` instance?